src/gutter-container.coffee

```diff
@@ -0,0 +1,100 @@
+{Emitter} = require 'event-kit'
+{Subscriber} = require 'emissary'
+Gutter = require './gutter'
+
+# This class encapsulates the logic for adding and modifying a set of gutters.
+
+module.exports =
+class GutterContainer
+  Subscriber.includeInto(this)
```

Comment:
You should use a `CompositeDisposable` from `event-kit` instead of `Subscriber` from `emissary`, which we're phasing out. I know we use it elsewhere but I'd rather not add any new uses.